code/modules/antagonists/nations/factions/cargo.dm

```diff
@@ -0,0 +1,19 @@
+/datum/team/nationcargo
+	name = "Cargonia"
+	member_name = "Cargonians"
+
+/datum/antagonist/cargonia
```

Comment:
Shouldn't this be a subtype of antagonist/nations? Along with all the others too?